server/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "server for Brave Buttons chatbot and heartbeat monitoring",
   "main": "server.js",
   "dependencies": {
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.5.0",
+    "@aws-sdk/client-iot-wireless": "^3.50.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-generateCalculatedTimeDifferenceString",
```

Comment:
Remember to update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/38 is merged

Reply:
Done